modules/Authentication.php

```diff
@@ -49,18 +77,12 @@ public static function withEmailPassword($db, $email, $password)
         return null;
     }
 
-    public function getUserIdFromSessionToken($session_token)
-    {
-        $stmt = $this->db->prepare("SELECT * FROM `user_session` WHERE `session_token` = ?");
-        $stmt->bindParams('i', $session_token);
-        $stmt->execute();
-        $result = $stmt->getResults();
-        $row = $result->fetchAssoc();
-        $stmt->close();
-        return $row;
-    }
-
-    // Create session token and adding log in database for validation
+    /**
+     * @method : createSession
```

Comment:
These `@method` seem redundant. @npari I saw you mentioned a commenting guide in Slack but in PHP we use `@method` only when creating class comments to specify the magic methods the class has access to ensure faster coding in IDE.